specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -555,13 +571,25 @@ def verify_cell_kzg_proof_batch(row_commitments_bytes: Sequence[Bytes48],
 ### `construct_vanishing_polynomial`
 
 ```python
-def construct_vanishing_polynomial(missing_cell_ids: Sequence[CellID]) -> Tuple[
-        Sequence[BLSFieldElement],
-        Sequence[BLSFieldElement]]:
+def construct_vanishing_polynomial(missing_cell_ids: Sequence[CellID]) -> Sequence[BLSFieldElement]:
     """
-    Given the cells that are missing from the data, compute the polynomial that vanishes at every point that
+    Given the cells IDs that are missing from the data, compute the polynomial that vanishes at every point that
     corresponds to a missing field element.
     """
+
+    # If all of the cells are missing, then the vanishing polynomial
+    # can be computed as Z(x) = x^n - 1, where `n` is FIELD_ELEMENTS_PER_EXT_BLOB
+    # 
+    # Note: this makes the function complete on all inputs, however this code path should not 
+    # be hit since this method is used for data recovery and if all of the
```

Comment:
Not a huge fan of this unreachable code. Not gonna die on this hill though. My comments on the thread below.